Moderation-Policy.md

```diff
@@ -192,20 +192,47 @@ of the [Code of Conduct][].
 * Moderation of Posts authored by non-Collaborators may result in those
   non-Collaborators being Blocked temporarily or indefinitely from further
   participation in the Node.js GitHub organization.
-* One may immediately bypass the above procedures if it is clear that there is no intention to
-  collaborate in good faith. In such cases, it is permissible to immediately:
-  * remove posts
-  * lock issues/pull requests
-  * delete issues/pull requests
-  * block users
-  * report accounts to GitHub
+* It is always permitted and encouraged to report users interacting in bad
+  faith to GitHub.
```

Comment:
No strong opposition to including this, but it seems superfluous and I've always wanted a short and scannable policy. So I'm going to suggest removing it.

```suggestion
```

Reply:
This was just here since it was referenced in the old policy. Happy either way